### ISSUE TITLE
Use of onedriver with nsswitch and autofs

### ISSUE THREAD

**Issue by dgtlrift** [jstaf/onedriver#251]
Is there any documentation describing the use of onedriver with nsswitch and autofs such that automounting users onedrive location?

**Comment by jstaf** [comment]
No, but if you enable the corresponding systemd user service, onedriver will automount on its own when the user logs in.
